app/addons/fauxton/assets/less/navigation.less

```diff
@@ -0,0 +1,177 @@
+  left: 0;
+  top: 0;
+  bottom: 0;
+  z-index: 5;
+
+}
+
+.faux-navbar__itemarea {
+  .box-sizing(border-box);
+  border-bottom: 1px solid @brandDark2;
+  height: 48px;
+  padding: 10px 20px;
+  line-height: 24px;
+}
+
+.faux-navbar__version-footer {
+  color: @buttonText;
+  font-size: 10px;
+  text-align: center;
+  background-color: @brandDark2;
+}
+
+.faux-navbar__burger {
+  background-color: @brandDark2;
+  padding: 19px 0 18px 18px;
+  position: fixed;
+  z-index: 100;
+  top: 0;
+}
+
+.faux-navbar--narrow {
+  width: @collapsedNavWidth;
+}
+
+.faux-navbar--wide {
+  width: @navWidth;
+}
+
+.faux-navbar__burger:hover .faux-navbar__burger__icon {
+  color: @navIconActive;
+}
+
+.faux-navbar__burger__icon {
+  color: @navIconColor;
+  font-size: 27px;
+}
+
+.faux-navbar__burger__icon--flipped:before{
+  -moz-transform: scale(-1, 1);
+  -webkit-transform: scale(-1, 1);
+  -o-transform: scale(-1, 1);
+  -ms-transform: scale(-1, 1);
+  transform: scale(-1, 1);
+}
+
+.faux-navbar__link, .faux-logout__link, .faux-login__link {
+  display: block;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.faux-navbar__link--active {
+  background-color: @brandHighlight;
+  text-decoration: none;
+}
+
+.faux-navbar__link--inactive {
+  background-color: @brandDark1;
+}
+
+.faux-navbar__link:hover, .faux-logout__link:hover, .faux-login__link:hover {
+  background-color: @hoverHighlight;
+  text-decoration: none;
+}
+
+.faux-navbar__link:active {
+  text-decoration: none;
+}
+
+.faux-navbar__link:hover .faux-navbar__icon:before {
+  color: @navIconActive;
+}
+
+.faux-navbar__link--active .faux-navbar__icon:before {
+  color: @navIconActive;
+}
+
+.faux-navbar__icon {
+  margin-right: 14px;
+  color: @navIconColor;
+  font-size: 24px;
+  vertical-align: middle;
+}
+
+.faux-navbar__text {
+  margin: 0;
+  color: @buttonText;
+  vertical-align: middle;
+  font-size: 16px;
+  font-weight: normal;
+  font-family: Helvetica,sans-serif;
+  font-weight: 400;
+}
+
+.faux-navbar__logout__text {
+  font-size: 12px;
+  color: @buttonText;
+}
+
+.faux-navbar__logout__textcontainer {
+  text-align: center;
+  color: @buttonText;
+}
+
+.faux-navbar__logout__textcontainer--narrow {
+  padding-bottom: 4px;
+  padding: 20px 0;
+}
+
+.faux-navbar__logout__textcontainer--wide {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  padding: 20px;
+}
+
+.faux-navbar__login__textcontainer {
+  text-align: center;
+  color: @buttonText;
+}
+
+.faux-navbar__login__textcontainer--narrow {
+  padding-bottom: 4px;
+  padding: 20px 0;
+}
+
+.faux-navbar__login__textcontainer--wide {
+  padding: 20px;
+}
+
+.faux-navbar__brand {
+  margin: 20px 0 20px 0;
+  height: 50px;
+  padding: 10px 10px 10px 10px;
+  float: none;
+  background: @brandDark2;
+}
+
+.faux-navbar__brand-logo {
+  display: block;
+  height: 100%;
+  margin-top: 10px;
+}
+
+.faux-navbar__brand-logo--wide {
+  background: url(../../../../../assets/img/CouchDB-negative-logo.png) no-repeat 23px 0px;
```

Comment:
It would be nice to set these logo's via our settings.json file. I know the pouchdb team would like to override these